src/classes/TEST_OpportunityBuilder.cls

```diff
@@ -0,0 +1,229 @@
+/*
```

Comment:
This is good, though I'd like to add a new method to handle custom fields. Something along the lines of 
```
public TEST_OpportunityBuilder customField(String fieldName, Object value) {
    valuesByFieldName.put(fieldName, value);
}
```